apps/nextra/docs.config.js

```diff
@@ -13,154 +13,300 @@ export const getOrigin = () => {
+  feePayerLabel: "Fee Payer",
+  feePayerDescription:
+    "Empower your transactions with seamless execution, letting others foot the bill",
+  multiSigLabel: "Multi-sig",
+  multiSigDescription:
+    "Secure your assets like never before with collaborative control and unparalleled safety",
+  gasLabel: "Gas",
+  gasDescription: "Trade smarter, not harder with low gas fees",
+  consensusLabel: "Consensus",
+  consensusDescription: "Learn more about DAG based consensus",
+  storageLabel: "Storage",
+  storageDescription:
+    "Learn more about Aptos' Jellyfish Merkle Tree and custom RocksDB configuration",
+  networkingLabel: "Networking",
+  networkingDescription:
+    "Learn about the topology of the Aptos blockchain and how nodes communicate with each other",
+  mempoolLabel: "Mempool",
+  mempoolDescription:
+    "Learn how transactions are buffered for processing before they're sent upstream",
+  stateSyncLabel: "State Sync",
+  stateSyncDescription:
+    "Learn how nodes downstream synchronizes with the latest state",
+
+  // Testimonials Section
+  testimonialsSectionHeadline: "Hear From The Ecosystem",
+  testimonial1Body:
+    "Building on Aptos has been transformative, offering unmatched transaction speed and reliability. We love the vibrant community and overall direction that the Aptos ecosystem is heading.",
+  testimonial1Author: "–Muhd Dahlan, CEO Zabava Labs",
+  testimonial2Body:
+    "Aptin Finance drives secure, intuitive, and seamless financial interactions for Aptos users. Aligned with Aptos' vision, we aim to establish a nexus of Omni-chain DeFi opportunities for a diverse global community.'",
+  testimonial2Author: "–Elvis Woo, CTO & Co-Founder, Aptin Finance",
+  testimonial3Body:
+    "Cellana Finance chose to build on Aptos because of its rapid transaction speed, cost-effectiveness, and robust security. Moreover, Aptos’s instant finality (less than 1s) will be a significant strength in attracting a variety of financial applications from Web 2 in the future.",
+  testimonial3Author: "–Andy Hoang, CEO, Cellana",
+
+  // Footer Section
+  aptosAlt: "Aptos Foundation",
+  discordAlt: "Discord",
+  githubAlt: "Github",
+  linkedinAlt: "LinkedIn",
+  telegramAlt: "Telegram",
+  twitterAlt: "Twitter",
+  buildHeading: "Build",
+  discoverHeading: "Discover",
+  connectHeading: "Connect",
+  meetAptosHeading: "Meet Aptos",
+  documentationLink: "Documentation",
+  governanceLink: "Governance",
+  networkNumbersLink: "Network Numbers",
+  validatorsLink: "Validators",
+  nodeOperationsLink: "Node Operations",
+  ecosystemHubLink: "Ecosystem Hub",
+  grantsLink: "Grants",
+  pressLink: "Press",
+  eventsLink: "Events",
+  collectiveLink: "Collective",
+  forumLink: "Forum",
+  aboutAptosLink: "About",
+  netZeroPolicyLink: "Net Zero Policy",
+  whitePaperLink: "White Paper",
+  brandLink: "Brand",
+  careersLink: "Careers",
+  privacyLink: "Privacy",
+  termsLink: "Terms",
+};
+
+export const zh = {
```

Comment:
We can destructure `en` in the beginning to reuse code better

```suggestion
export const zh = {
  ...en,
  locale: "zh",
  name: "简体中文",
  direction: "ltr",
  title: "Aptos 文档",
  editText: "在 Github 上修改当前页面 →",
  feedbackText: "找到问题? 请给我们反馈 →",
  footerLinkText: "https://vercel.com/?utm_source=swr",
  footerLinkElement: undefined,
  ...
```

Reply:
Done